evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -43,7 +43,7 @@
         <span class="fas fa-comment icon-gray" data-toggle="tooltip" data-placement="top" title="{% trans 'No text answers available' %}"></span>
     </td>
 {% elif evaluation.num_textanswers > evaluation.num_reviewed_textanswers %}
```

Comment:
The predicate in the template is semantically connected to `text_answer_review_is_urgent` – as far as I understand, it currently can be true in cases where there are no text answers.
How do you feel about adding some more properties to reflect the text answer review state? Or change the property to return an enum `{NO_TEXTANSWERS, REVIEW_NEEDED, REVIEW_URGENT, REVIEWED}`? This would also remove more logic from the template (yay)

Reply:
Done. However, its 100 lines more code (as the testing is quite intensive now). I don't know, I'm still unsure about where the perfect balance between testing and productive code is - I think this one is too far on the testing side now...